### ISSUE TITLE
Request for v1.0.8 [or more recent] updates

### ISSUE THREAD

**Issue by whorfin** [pipewire-debian/pipewire-debian#111]
The project splash page notes:
```
An upstream version of PipeWire for Ubuntu maintained with a 15 day release cycle
```

The PPA seems stuck on v1.0.7 though, while upstream is on to 1.2.6/1.0.9
A v1.2.3 set of changes, backported to v1.0.8, are of keen interest to me as they may help with FFADO issues.

I'm very appreciative of this project as it has allowed greatly improved pipewire and wireplumber experiences for Ubuntu, so I am hesitant to log this as a nag, more a question as to roadmap of the project and a query as to the intent behind the release cycle

Cheers

**Comment by user3354** [comment]
If you are using Ubuntu LTS oder Linux Mint, you can head over to Rob Savory's Repository.

**Comment by paranoidi** [comment]
PSA: Do not simply add Robs PPA and expect it to take over nicely. I have ran into a dependency mess that I don't think I will be able to get out.

**Comment by user3354** [comment]
Yes, always use a test system with new PPAs. I use Rob's PPA with success on half a dozen systems (Mint and Ubuntu Studio).
